### DISCUSSION TITLE
LINK SUBMISSION

### DISCUSSION THREAD

**Discussion by Clin-Tech** [zuri-training/Fetch_Metadata_Team82#43] [General]
Hello mates, please I want everyone to check if this link isn`t displaying 404 link the link we submitted during our first presentation.

https://github.com/zuri-training/Fetch_Metadata_Team82

**Comment by iamelovia** [comment]
https://github.com/zuri-training/Fetch_Metadata_Team82

The link above opened on my incognito tab
